src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,340 @@
+    "--requirements_file",
+    default=None,
+    type=str,
+    help="""Optional created virtual environment to install with the
+libraries specified in this requirements file. If not, the activated virtual environment is
+cloned through a temporary ``pip list`` file. If you are using an editable package,
+you should attach your own requirement file using ``pip freeze`` """,
+)
+@click.option(
+    "--extra_files",
+    default=None,
+    type=str,
+    help="""Extra files to upload that can be called from your main Python file
+(or from the shell file).""",
+)
+@click.option(
+    "--config_file",
+    default=None,
+    type=str,
+    help="""File to load the job configuration from.""",
+)
+@click.option(
+    "--save_config_file",
+    default=False,
+    type=bool,
+    is_flag=False,
+    flag_value=True,
+    help="""
+Whether to write the configuration to the configuration file (specified
+using the ``config_file`` argument) after the job has been successfully submitted.
+The default is ``False``. If ``True``, and the file already exists, the configuration file is overwritten.""",
+)
+@click.option(
+    "--num_cores",
+    default=None,
+    type=str,
+    help="""Number of CPU cores reserved for the job. The default is ``1``""",
+)
+@click.option(
+    "--memory",
+    default=None,
+    type=str,
+    help="""Amount of memory (RAM) in MB reserved for the job. The default is ``100 MB``.""",
+)
+@click.option(
+    "--disk_space",
+    default=None,
+    type=str,
+    help="""Amount of hard drive space in MB reserved for the job. The default is ``100 MB``.""",
+)
+@click.option(
+    "--exclusive",
+    default=None,
+    type=str,
+    is_flag=False,
+    flag_value=True,
+    help="""Whether the job is to run on a machine that is running no other jobs running. The default is ``False``.""",
+)
+@click.option(
+    "--max_execution_time",
+    default=None,
+    type=int,
+    help="""Maximum execution time for the job. The default is zero (unlimited).""",
+)
+@click.option(
+    "--wait",
+    default=None,
+    type=str,
+    is_flag=False,
+    flag_value=True,
+    help="""Whether the terminal is to wait for job completion before returning control to the user. """,
+)
+@click.option(
+    "--mode",
+    default=None,
+    type=str,
+    help="""
+Force the job submission to behave as if the main file was a Python,
+shell, or APDL file, regardless of its extension type. Allowed values are
+``"python"``, ``"shell"``, and ``"apdl"``.
+By default, PyMAPDL detects the type of file from its extension.
+""",
+)
+@click.option(
+    "--output_to_json",
+    default=None,
+    type=str,
+    is_flag=False,
+    flag_value=True,
+    help="""Print the output values to the terminal as json. It requires to use ``--wait`` value too. """,
+)
+@click.option(
+    "--debug",
+    default=False,
+    type=bool,
+    is_flag=False,
+    flag_value=True,
+    help="""Whether PyMAPDL is to print debug logging to the console output.""",
```

Comment:
```suggestion
    help="""Whether PyMAPDL is to print debug logging to the console.""",
```
It doesn't seem output is needed.